lib/test_queue/runner.rb

```diff
@@ -337,6 +349,7 @@ def distribute_queue
               # If we have a slave from a different test run, don't respond, and it will consider the test run done.
               sock.write("OK\n")
               remote_workers += num
+              quorum_members += num
```

Comment:
It is decremented when workers disconnect. In theory that only happens when the test run ends but it seemed a little clearer to track "number of workers that have ever connected" separately.